### ISSUE TITLE
Revise definition of Diversity in the Glossary

### ISSUE THREAD

**Issue by jenstrickland** [w3c/PWETF#342]
@TzviyaSiegman, @wareid, and I collaborated on a revised definition for diversity in the glossary.

We aimed for a definition and then a separate list of examples.

## Current definition
> Diversity for the purposes of diversity and inclusion, is any dimension that can be used to differentiate groups and people from one another. Dimensions can include but are not limited to:  socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.

## Proposed revision
Diversity is the consideration of a wide range of varying identities, lived experiences, and perspectives.

### Separate list of dimensions
This range can include, but is not limited to: socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.

**Comment by fantasai** [comment]
Diversity isn't a form of consideration, so I don't think you can define it as "consideration" (of anything). But (stealing from your use of "varying"), you could define it as "variation", so maybe something like:

> Diversity, for a group of people, is variation across a wide range of identities, lived experiences, abilities, and perspectives.

**Comment by wareid** [comment]
I'm happy with the wording change on the main definition, seems fine to me. 

I would keep the second part as: 

> This can include, but is not limited to: ...

**Comment by dbooth-boston** [comment]
I like these suggestions, though I think 'for a group of people" is not needed:

> Diversity is variation across a wide range of identities, lived experiences, abilities, and perspectives.

Also, I suggest dropping "indigeneity" only because it is not a well-known word -- apparently coined very recently, and not in most dictionaries -- and we already have immigration status, ethnicity and nationality listed:

> This can include, but is not limited to: socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.

**Comment by a11ydoer** [comment]
I would be fine with "perceived race" upon the suggestion by @dbooth-boston at PWE meeting instead of "ascribed race" if it present the external aspect of "race" which is juxtaposing to the internal aspect of "racial identity"

**Comment by nigelmegitt** [comment]
One concern I would have with the term "perceived race" is that it appears to diminish the importance of ascribed or identified race that is difficult to perceive, but where there is nevertheless a strong possibility or history of discrimination. For example even if the external perception of race is primarily based on skin colour, being a highly visible characteristic, it would be equally unacceptable to discriminate against one "race" with the same skin colour as another. By focusing on perception this point is diluted.

**Comment by nigelmegitt** [comment]
> what if we use both "perceived race" and "racial identity" instead of "race" and "racial identity"?

That's exactly what I think we should _not_ do.

> racial discrimination is necessarily based on the discriminating party's perception

I disagree. It's based on what the discriminating party understands the other's race to be, not necessarily what they perceive it to be. For example, that understanding could come from someone else telling them the other's race.

**Comment by jenstrickland** [comment]
The world uses "race" doesn't it?

**Comment by wareid** [comment]
I mentioned this during the meeting, but I have never seen an instance in a definition (and at this point, I have seen too many), where when race is mentioned, there is a modifier of any kind. We're already breaking the mold a bit with the addition of "racial identity", but it is something people have seemed comfortable with. 

At this point we're bikeshedding common terms, which seems counter-productive to the goal of creating a more inclusive document.

**Comment by chaals** [comment]
I realise the group has closed this issue. As input from another person (that brings this to 7 people) which I would provide in a review, I am concerned that the discussion has closed too fast, potentially landing on the wrong outcome.

I am concerned about the use of "race", for the reasons Florian outlined, that in many cases beyond english it is taken to mean a biological difference and that it is not (even when not considered to be biologically based) universally recognised as a thing.

I believe *the world* does not use "race". It is my perception that big chunks of the world are very uncomfortable with that as a term.

I would be strongly in favour of @dbooth-boston's proposed "perceived race or racial identity", which I think provides clarity. It also provides for the case where an individual claims a "racial identity" (and in very large part avoids the issues that can arise around whether others believe that claim or not).

With respect to [@nigelmegitt's argument against it](#issuecomment-1700652581), my perception is that what someone perceives another person's race to be and what they understand it to be generally amount to the same thing: A perception about a character in a book, who is never ascribed a 'race', can arise from many different ways including the way the speak or the things they speak about or conversations with real friends or newspaper articles such as book reviews talking about the character's 'race'. 

In other words, I perceive the words "understood", "ascribed" and "perceived" as adjectives for 'race' to have the same effective meaning, and of those "perceived" to be the one that seems most normal and easily understood.

**Comment by jenstrickland** [comment]
We cannot talk about racism without race. 

Where in the world do they **not** use race?

If we could include representatives from those parts of the world, that would help. 

There aren't many of us who experience racism in this discussion. I'd like to include more of us.

**Comment by frivoal** [comment]
> We cannot talk about racism without race.

This is not a question about racism. The parts of the CEPC that condemn racism—and the usage of the word race there—has not been challenged.

This is a question of the usage of the word race as one of the things whose diversity we promote.
